Branch168 Need Model

francisbetikoetxea opened the pull request:
Edited Need model to reflect the Need object as defined per the new database schema in #163.

Attempts to complete issue #168

francisbetikoetxea committed:
Preliminary commit for fixing needs table

francisbetikoetxea committed:
Merge branch 'develop' into branch163NeedTable

Tiernebre commented on donation-api/src/main/java/org/interfaithsanctuary/donationapi/model/Need.java:
```diff
@@ -1,45 +1,28 @@
 
     @Id
-    @Column(name = "need_id")
+    @Column(name = "needId")
```
I should have clarified in the task, the column annotation names still use the need_id case, for the column names that are being used in the new database table. It's the Java instance variables themselves that should follow the syntax I specified. That's on me, I can go and fix this if you want me to!

francisbetikoetxea committed:
Update Need.java

Reverted the column names to their previous format. I did change the "unit" variable to reflect more precisely what it is. I think that "unit of measurement" lessens the confusion between what a "unit" can mean (either a numeric quantity or a unit of measure).

Tiernebre reviewed (approved):
Looks good! Thanks for fixing the column names!
